src/commands/append/fact.ts

```diff
@@ -0,0 +1,77 @@
+import {Flags} from '@oclif/core'
+import {TokenAwareCommand} from '../../lib/token-command'
+import {initClient} from '../../lib/client'
+import fs from 'node:fs/promises'
+import JSON5 from 'json5'
+import Ajv from 'ajv'
+
+export default class Reset extends TokenAwareCommand {
+  static description = 'Reset a Ledger'
+
+  static examples = [
+    `$ evently append:fact <<EVENT 
+{
+  entity: "thermostat",
+  event: "temperature-recorded",
+  key: "thermostat1",
+  meta: {causation: "14rew3494"},
+  data:{celsius: 18.5}
+}
+EVENT
```

Comment:
Instead of using an argument, I've opted to use STDIN here. I feel this is is a more natural way to use this, given that people are unlikely to hand-write this complicated JSON body on the command line on every use.

As you said this is more likely to be part of some automated processing. We _could_ optionally still have another argument that lets you read straight from the command line (like `--input` maybe?)

Reply:
Interesting, that makes sense.